addons/website_sale/views/templates.xml

```diff
@@ -555,7 +555,7 @@
     </template>
 
     <template id="base_unit_price" name="Product Bas eunit price">
-        (<span class="o_base_unit_price" t-field="product.base_unit_price" t-options="{'display_currency': website.currency_id}"/>
+        (<span class="o_base_unit_price" t-esc="round(combination_info['base_unit_price'],2)"/> <span t-esc="website.currency_id.symbol"/>
```

Comment:
This removes compatibility with currencies which display the symbol on the left, it seems like we can use `t-out` along with `t-options-widget` instead.